<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParkManagement</class>
 <widget class="QMainWindow" name="ParkManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1444</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParkManagement</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ParkManagementSystem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Entence">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>450</y>
      <width>811</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ParkAreaDown"/>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>811</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ParkAreaUp"/>
   </widget>
   <widget class="QLCDNumber" name="freeArea">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>64</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FreeArea</string>
    </property>
   </widget>
   <widget class="QPushButton" name="entenceVerify">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitVerify">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>360</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RandomCar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RandomCar</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="movementCar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>movement</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1444</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
